### PR TITLE
Hide duplicate on relational fields

### LINKED ISSUE #498: Disable "Duplicate Field" for the PK and Relational Fields
We could duplicate M2Os, but would need to also duplicate the relationship.

### PR DESCRIPTION
**Pull request by Nitwel**
closes #498

### CODE REVIEW

File: app/src/modules/settings/routes/data-model/fields/components/field-select.vue [hunk 1]
```diff
@@ -90,7 +90,7 @@
 								</v-list-item-content>
 							</v-list-item>
 
-							<v-list-item @click="duplicateActive = true">
+							<v-list-item v-if="duplicateable" @click="duplicateActive = true">
```
Comment (rijkvanzanten) [inline app/src/modules/settings/routes/data-model/fields/components/field-select.vue:93]: duplicatable*

File: app/src/modules/settings/routes/data-model/fields/components/field-select.vue [hunk 2]
```diff
@@ -288,13 +297,21 @@ export default defineComponent({
 			);
 			const duplicateTo = ref(props.field.collection);
 
+			const duplicateable = computed(() => {
+				return (
+					['o2m', 'm2m', 'm2o', 'files', 'file'].includes(props.field.type) === false &&
```
Comment (rijkvanzanten) [inline app/src/modules/settings/routes/data-model/fields/components/field-select.vue:302]: + m2a

File: app/src/modules/settings/routes/data-model/fields/components/field-select.vue (hunk 1 above)
Comment (benhaynes) [reply app/src/modules/settings/routes/data-model/fields/components/field-select.vue:93]: I prefer duplicable*

Comment (rijkvanzanten) [reply app/src/modules/settings/routes/data-model/fields/components/field-select.vue:93]: works too, as long as it's correctly spelled :grin

Comment (Nitwel) [reply app/src/modules/settings/routes/data-model/fields/components/field-select.vue:93]: Whopsie Haha.

**Review by rijkvanzanten — APPROVED** [review]

**Pull request merged**